c10/core/DispatchKeySet.h

```diff
@@ -61,8 +61,8 @@ class DispatchKeySet final {
     }
   }
   // Test if a DispatchKey is in the set
-  bool has(DispatchKey t) const {
-    TORCH_INTERNAL_ASSERT(t != DispatchKey::Undefined);
+  bool inline has(DispatchKey t) const {
```

Comment:
nit: `inline` here has no effect; defining a method in the class definition already makes it inline. See the second paragraph of https://en.cppreference.com/w/cpp/language/inline